esm/svg/element.js

```diff
@@ -21,6 +23,19 @@ export class SVGElement extends Element {
     this.ownerSVGElement = ownerSVGElement;
   }
 
+  get viewBox() {
```

Comment:
I am not sure this is how Proxy handler works ... this should likely be a `get(target, name)` variant where you check for the `name` ???